spyder/widgets/fileswitcher.py

```diff
@@ -609,12 +615,16 @@ def setup_file_list(self, filter_text, current_path):
         # Get optional line number
         if trying_for_line_number:
             filter_text, line_number = filter_text.split(':')
+            # Get all the available filenames
+            scores = get_search_scores('', self.filenames,
+                                       template="<b>{0}</b>")
```

Comment:
I did this in order to show all the options with their respective line count. It could be more clear with the preview below.